.coveragerc

```diff
@@ -48,6 +48,7 @@ omit =
     homeassistant/components/downloader.py
     homeassistant/components/keyboard.py
     homeassistant/components/light/hue.py
+    homeassistant/components/light/mqtt.py
```

Comment:
This one should not be there. We have test coverage for all components that build on top of MQTT. If you want to give an initial stab at writing tests, see the [MQTT switch tests](https://github.com/balloob/home-assistant/blob/dev/tests/components/switch/test_mqtt.py).